src/shared/sam/debugger/awsSamDebugConfigurationProvider.ts

```diff
@@ -1,70 +0,0 @@
-/*!
- * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import * as vscode from 'vscode'
-import { CloudFormationTemplateRegistry } from '../../cloudformation/templateRegistry'
-
-import { localize } from '../../utilities/vsCodeUtils'
-import { AwsSamDebuggerConfiguration, createAwsSamDebugConfigurationForTemplate } from './awsSamDebugConfiguration'
```

Comment:
createAwsSamDebugConfigurationForTemplate appears to have been replaced with createDirectInvokeSamDebugConfiguration. Can the createAwsSamDebugConfigurationForTemplate function be removed?